src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -626,6 +624,8 @@ public void configureWebApp() throws Exception {
             FileUtils.deleteDirectory(extractedWebAppDir);
         }
         getWebAppConfig().setWar(webAppFile.getCanonicalPath());
+        // allow Jetty to accept a bigger form so that it can handle update center JSON post
```

Comment:
The comment needs to state the larger reason why max form content size is adjusted
```suggestion
        // allow Jetty to accept large forms because Jenkins accepts large forms at runtime
        // Jenkins cloud configuration is an example of a case where large forms are needed.
```